meson.build

```diff
@@ -55,10 +66,6 @@ conf_data.set('SYSCONFDIR', join_paths('/', get_option('sysconfdir')))
 add_project_arguments(['-DWLR_USE_UNSTABLE'], language: ['cpp', 'c'])
 add_project_link_arguments(['-rdynamic', '-Wl,-E'], language: 'cpp')
 
-ccp = meson.get_compiler('cpp')
-wlroots_has_xwayland = ccp.get_define('WLR_HAS_XWAYLAND', prefix: '#include <wlr/config.h>', dependencies: wlroots) == '1'
-wlroots_has_x11_backend = ccp.get_define('WLR_HAS_X11_BACKEND', prefix: '#include <wlr/config.h>', dependencies: wlroots) == '1'
-
```

Comment:
It would be nice to see these in the configure output if get_option('use_system_wlroots'). Can we output these with the rest of the variables conditionally dependent on get_option('use_system_wlroots')?

Reply:
fixed